README.md

```diff
@@ -1,53 +1,804 @@
-#SF-WDI-23 Homework submission
 
-Homework will be turned in to your fork of this repository daily.
 
-##Setup
 
-To set up your initial homework repository do the following:
```

Comment:
not sure what happened here.... 

it looks like you somehow pasted an html document into the README.md. :dromedary_camel: